src/config-utils.ts

```diff
@@ -17,11 +17,17 @@ export class ExternalQuery {
     }
 }
 
+// The set of acceptable values for built-in suites from the codeql bundle
+const builtinSuites = ['security-experimental', 'security-and-quality'] as const;
```

Comment:
```suggestion
const builtinSuites = ['security-extended', 'security-and-quality'] as const;
```

Let's go with security-extended instead, since it's not strictly experimental, but just more queries of varied quality. Thanks!

Reply:
I should have said the suite names in this PR were mostly just a placeholder as I thought that decision was still being made. I'll update this once we know what they'll be.